open_humans/templatetags/utilities.py

```diff
@@ -217,19 +216,6 @@ def active(context, pattern_or_urlname):
     return ""
 
 
-@register.simple_tag()
-def url_slug(label):
```

Comment:
I guess the `source_to_url_slug` is also no longer needed... (not a blocker though -- a random spot check, now I wonder what other templatetags exist that are obsolete)

Reply:
Yeah, I'll have a look at those at some point.